EZSwiftExtensionsTests/DateTests.swift

```diff
@@ -229,67 +241,55 @@ class DateTests: XCTestCase {
     }
     
     func testEra() {
-        let customDate = Date(fromString: "12-01-2015 05:45:12", format: self.format)
-        XCTAssertEqual(customDate?.era, Calendar.current.component(Calendar.Component.era, from:customDate!))
+        XCTAssertEqual(date.era, Calendar.current.component(Calendar.Component.era, from: date))
     }
     
     func testYear() {
-        let customDate = Date(fromString: "12-01-2015 05:45:12", format: self.format)
-        XCTAssertEqual(customDate?.year, 2015)
+        XCTAssertEqual(self.date.year, 2015)
     }
     
     func testMonth() {
-        let customDate = Date(fromString: "09-01-2015 05:45:12", format: self.format)
-        XCTAssertEqual(customDate?.month, 1)
+        XCTAssertEqual(self.date.month, 11)
     }
     
     func testMonthAsString() {
-        let customDate = Date(fromString: "09-01-2015 05:45:12", format: self.format)
-        XCTAssertEqual(customDate?.monthAsString, "January")
-        
-        let customDate2 = Date(fromString: "10-03-2009 05:45:12", format: self.format)
-        XCTAssertEqual(customDate2?.monthAsString, "March")
-        
-        let customDate3 = Date(fromString: "11-11-2012 05:45:12", format: self.format)
-        XCTAssertEqual(customDate3?.monthAsString, "November")
+        XCTAssertEqual(self.date.monthAsString, "November")
     }
     
     func testWeekDay() {
-        let customDate = Date(fromString: "09-01-2015 05:45:12", format: self.format)
-        XCTAssertEqual(customDate?.weekday, "Friday")
+        XCTAssertEqual(self.date.weekday, "Sunday")
+        NSTimeZone.default = TimeZone(abbreviation: "UTC")!
```

Comment:
I am setting a default timmezone because the tests are failing for me because Im on another time zone. So the tests on `day`, `hour` etc variables of the Date extension are failing.

I thought that forcing a default timezone to control the setup of the environment is the way to go.

Please take a look at `testHour()` and `testDay()` test cases. The results differ when the default time zone changed and I am thinking this makes the tests better.

What do you think?